src/gui/options_banlist.h

```diff
@@ -0,0 +1,29 @@
+#ifndef OPTIONS_BANLIST_H
+#define OPTIONS_BANLIST_H
+
+#include <QDialog>
+
+namespace Ui
+{
+    class options_banlist;
+}
+
+class options_banlist: public QDialog
+{
+    Q_OBJECT
+
+public:
+    explicit options_banlist(QWidget *parent = 0);
```

Comment:
Can not give any rationale except that it is autogenerated by Qt creator code. And of course I ignored that parameter when instantiated object that class. I read documentation and and found that NULL parameter is fully legal for QDialog, but creates distinct taskbar entry for window of dialog. So now I pass a parent when creating an object.